startMitm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I'm not quite sure what the purpose of this file is. Is it just for demonstration/informational purposes? That might be better in the README.md file.